### PR TITLE
update fix sd bug

### PR DESCRIPTION
**Pull request by Derekwan1081546**

### CODE REVIEW

**Comment by Derekwan1081546** [comment]
update fix sd bug

File: package.json
Comment (seanwascoding) [inline package.json:1]: this file should be remove

File: package-lock.json
Comment (seanwascoding) [inline package-lock.json:1]: this file should be remove

**Comment by Derekwan1081546** [comment]
fix bug

**Comment by Derekwan1081546** [comment]
fix bug

**Comment by seanwascoding** [comment]
"node_modules" also need to be remove from outside the "server" folder.

**Comment by Derekwan1081546** [comment]
update fix bug

**Pull request merged**